openspec/changes/archive/2026-04-05-workspace-trigram-index/verify-report.md

```
@@ -0,0 +1,129 @@
+## Verification Report
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)